uclouvain-faculte-de-droit.csl

```diff
@@ -0,0 +1,558 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" default-locale="fr-BE" and="text" et-al-min="5" et-al-use-first="1" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>UCLouvain Faculté de droit</title>
```

Comment:
@adam3smith, should at least have a hyphen after the institution name. Per https://uclouvain.be/fr/instituts-recherche/juri/cedie, probably `<title>UCLouvain - CeDIE  (French)</title>` or `<title>UCLouvain - Centre Charles De Visscher pour le droit international et européen (French)</title>`

Reply:
(@adam3smith, let me know if you have a preference and I'll make the change)

Reply:
@anfourny is this style for the whole law faculty or "just" for CeDIE?

Reply:
@adam3smith this was originally created within, and for, the CeDIE research center but other centers within the faculty of law have shown interest, which is why I had used a broader title. But I think you're right to suggest the use of the name of our center. In that case, I find the full name as suggested by @rmzelle `<title>UCLouvain - Centre Charles De Visscher pour le droit international et européen (French)</title>`  a better option since "CEDIE" is not that self-explanatory.
Thanks again!

Reply:
Thanks! @rmzelle you'll make that change?

Reply:
Done, thanks @anfourny, @adam3smith.

https://github.com/citation-style-language/styles/pull/4017